scripts/compare_strategies.py

```
@@ -0,0 +1,334 @@
+        scoring_mode = report["scoring_mode"]
+        scores_b = report["results"]["strategy_b"]
+
+        if baselines_data is None:
+            baselines_data = {"schema_version": 1}
+
+        if strategy_b_id not in baselines_data:
+            baselines_data[strategy_b_id] = {}
+
+        # Preserve old entry under a superseded key
+        if scoring_mode in baselines_data[strategy_b_id]:
+            date_str = datetime.now().strftime("%Y-%m-%d")
```

Ruff (0.15.10)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)